images/capi/packer/config/kubernetes.json

```diff
@@ -12,9 +12,9 @@
   "kubernetes_rpm_gpg_check": "True",
   "kubernetes_rpm_gpg_key": "\"https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg\"",
   "kubernetes_rpm_repo": "https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64",
-  "kubernetes_rpm_version": "1.21.9-0",
-  "kubernetes_semver": "v1.21.9",
-  "kubernetes_series": "v1.21",
+  "kubernetes_rpm_version": "1.20.14-0",
```

Comment:
Is this change specific to us?

Reply:
yes , incorrect checkin . need to back this out.

Reply:
Reverted